TASK-5/Menu.cs

```diff
@@ -0,0 +1,202 @@
+                    ShowListOfAllCustomers(users);
+                    Console.Write("Ваш выбор: ");
+                    _selectedNumberOfCustomer = Convert.ToInt16(Console.ReadLine());
+
+                    var customerWares = ShowWaresOfADefiniteCustomer(_selectedNumberOfCustomer, users);
+                    ShowTotalSumOfWares(_line, customerWares);
+
+                    break;
+
+                case 3:
+                    var newListOfCustomers = AddNewCustomerToList(users);
+                    users = newListOfCustomers;
+
+                    break;
+
+                case 4:
+                    Console.WriteLine(
+                        "Введите номер покупателя, которому Вы хотите добавить/удалить товар из корзины: ");
+                    ShowListOfAllCustomers(users);
+                    Console.Write("Ваш выбор: ");
+                    _selectedNumberOfCustomer = Convert.ToInt16(Console.ReadLine());
+
+                    Console.Write("Добавить (1) или удалить (2)? ");
+                    var addOrRemove = Convert.ToInt16(Console.ReadLine());
+                    switch (addOrRemove)
+                    {
+                        case 1:
+                            AddWare(wares, users, _selectedNumberOfCustomer);
+
+                            break;
+
+                        case 2:
+                            RemoveWare(users, _selectedNumberOfCustomer);
+
+                            break;
+
+                        default:
+                            Console.WriteLine("\nНеизвестный номер.");
+
+                            break;
+                    }
+
+                    break;
+
+                case 0:
+                    Console.WriteLine("Выхожу...");
+
+                    break;
+
+                default:
+                    Console.WriteLine("\nНеизвестный номер.");
+
+                    break;
+            }
+        } while (_menuChoice != 0);
+    }
+
+    private static IEnumerable<Customer> AddNewCustomerToList(IEnumerable<Customer> users)
+    {
+        Console.Write("Введите номер паспорта: ");
+        var passportNumberOfCustomer = Console.ReadLine();
+
+        Console.Write("Введите имя покупателя: ");
+        var nameOfCustomer = Console.ReadLine();
+
+        Console.Write("Введите возраст покупателя: ");
+        var ageOfCustomer = Convert.ToInt16(Console.ReadLine());
+
+        var newCustomer = new Customer();
+        newCustomer.SetCustomer(passportNumberOfCustomer, nameOfCustomer, ageOfCustomer, new Cart());
+
+        var found = users.Where(u => u.Equals(newCustomer)).Count() > 0;
+        if (found)
+        {
+            Console.ForegroundColor = ConsoleColor.Red;
+            Console.WriteLine("Покупатель не добавлен, т.к. такой уже существует.\n");
+            Console.ResetColor();
+        }
+        else
+        {
+            users.Append(newCustomer);
+            Console.WriteLine("Добавлен новый пользователь.\n");
```

Comment:
Not working. Change IEnumerable to List, and method add() here

Reply:
Done.